docs/reference/plugins.md

```diff
@@ -1,12 +1,29 @@
 # Plugins
 There are a number of plugins available for pynxtools that are maintained within FAIRmat. These are extensions of pynxtools used for reading data of specific experimental techniques.
 
-- [**pynxtools-mpes**](https://github.com/FAIRmat-NFDI/pynxtools-mpes): A reader for multi-dimensional photoelectron spectroscopy data.
-- [**pynxtools-stm**](https://github.com/FAIRmat-NFDI/pynxtools-stm): A reader for scanning tunneling microscopy (SPM) and spectroscopy (STS) data.
-- [**pynxtools-xps**](https://github.com/FAIRmat-NFDI/pynxtools-xps): A reader for X-ray photoelectron spectroscopy (XPS) data. Documentation can be found [here](https://fairmat-nfdi.github.io/pynxtools-xps/).
+### Photoemission spectroscopy
+- [**pynxtools-mpes**](https://github.com/FAIRmat-NFDI/pynxtools-mpes): A reader for multi-dimensional photoelectron spectroscopy (MPES) data.
+- [**pynxtools-igor**](https://github.com/FAIRmat-NFDI/pynxtools-igor): A reader for MPES data stored in the [Igor Binary Wave Format](https://www.wavemetrics.com/).
```

Comment:
This repo does not exist. I have https://github.com/domna/pynxtools-igor but this just dumps some old stuff I worked on. The igor reader is basically included in the mpes and xps readers.

Reply:
removed